tests/testthat/test-z-queue.R

```diff
@@ -0,0 +1,24 @@
+context("integration: queue")
+
+test_that("run a job in a queue", {
+  workdir <- tempfile()
+  version <- r_version2()
+
+  q <- queue$new(version, workdir)
+  id <- q$submit(version, "reside-ic/defer", NULL)
+
+  w <- worker_create(workdir, NULL)
+  w()
```

Comment:
It would be cool to see this working end-to-end with a call to `pb_worker` and the repeat block but I guess that is pretty hard to test around. I think this combined with the interaction test for `pb_worker` is good.